litmus-portal/frontend/src/components/Header/ProfileDropDown.tsx

```diff
@@ -86,20 +86,26 @@ const ProfileDropdown: React.FC = () => {
             <div
               className={`${classes.profileDropdownRow} ${classes.profileUnset}`}
             >
-              <Typography id="emailUnset">
+              <Typography className={classes.emailUnset}>
                 {t('header.profileDropdown.emailUnset')}
               </Typography>
-              <Link
-                to={{
-                  pathname: '/settings',
-                  search: `?projectID=${projectID}&projectRole=${projectRole}`,
-                }}
-                onClick={() => tabs.changeSettingsTabs(0)}
-              >
-                <Typography title="Go to settings">
-                  {t('header.profileDropdown.emailSet')}
+              {projectRole === 'Owner' ? (
+                <Link
+                  to={{
+                    pathname: '/settings',
+                    search: `?projectID=${projectID}&projectRole=${projectRole}`,
+                  }}
+                  onClick={() => tabs.changeSettingsTabs(0)}
+                >
+                  <Typography title="Go to settings">
+                    {t('header.profileDropdown.emailSet')}
+                  </Typography>
+                </Link>
+              ) : (
+                <Typography className={classes.projectRoleHint}>
+                  Please switch to a project you own to access settings
```

Comment:
translation

Reply:
Translations for analytics will go as a separate as discussed.

Reply:
This is not for analytics, it's a header component file @Saranya-jena